### DISCUSSION TITLE
NEW: Improvements to the advanced tools center

### DISCUSSION THREAD

**Discussion by elizafarley** [amzn/ads-advanced-tools-docs#75] [Announcements]
Over the past couple of months, we've been making [small improvements](https://advertising.amazon.com/API/docs/en-us/info/release-notes#api-specifications-are-now-indexed-in-advanced-tools-center-search) to the advanced tools center website, and today we're announcing the release of our [new homepage](https://advertising.amazon.com/API/docs/en-us/index)!

From the new homepage you can:

* Easily access video tutorials, courses, user guides, and events. 
* Quickly navigate to popular content and see what’s new.
* Compare our different advanced tools offerings from a single page.

Stay tuned for more updates to the site in the coming months.

In the meantime, we'd love to hear any feedback or suggestions for how we can improve the site.

**Comment by AdsAPIPM** [comment]
Would love to hear what developers think about the new home page!

**Comment by elizafarley** [comment]
#### Another update: new search experience in the advanced tools center

We’ve introduced a new search UI to improve discoverability in the [advanced tools center](https://advertising.amazon.com/API/docs/en-us/index). The new design makes it easier to sort based on the type of content you’re looking for: Guides & tutorials, API reference documentation, or release notes. You can also see your past searches. 

We’ve also improved our search algorithm to cover more content and enabled fuzzy match, so even if your search term doesn’t match exactly, you should see results. 

Let us know if you have any feedback!

**Comment by elizafarley** [comment]
**New look and navigation for the advanced tools center**

We’ve released a new user interface for the advanced tools center, featuring an updated look and feel as well as a new navigation. The navigation is now responsive, and readable on mobile devices. 

Documentation is split into categories based on use case:

* [Developer guides](https://advertising.amazon.com/API/docs/en-us/guides/overview)—featuring tutorials, product overviews, and code samples for developers
* [API reference](https://advertising.amazon.com/API/docs/en-us/reference/api-overview)—containing OpenAPI specifications and conceptual content about REST APIs
* [No-code tools](https://advertising.amazon.com/API/docs/en-us/bulksheets/no-code-overview)—including bulk operations documentation

You can also find a dedicated section for [Support](https://advertising.amazon.com/API/docs/en-us/support/overview) and [Release notes](https://advertising.amazon.com/API/docs/en-us/release-notes/index) for all advanced tools.

If you have any feedback or other changes you'd like to see in the advanced tool center, please let us know!

**Comment by elizafarley** [comment]
**Improved OpenAPI specification rendering**

We have updated our rendering UI for OpenAPI specifications to provide an easier browsing experience for developers. The new layout allows easier navigation across the different methods and API endpoints. With this change, we have added a consistent layout across the schemas, fixed deep indentation when expanding and collapsing the response schemas, added the ability to view schemas and examples at the same time, and enabled easy copying of samples and parameters. API reference pages are also now mobile friendly and accessible. 

To explore the new layout, check out any of our [API reference documents](https://advertising.amazon.com/API/docs/en-us/reference/openapi-download). If you have any feedback, we'd love to here it!
